sdk/batch/azure-compute-batch/src/test/resources/session-records/JobTests.canCRUDJobWithPoolNodeCommunicationMode.json

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
We should also consider migrating the test recordings to assets repo.
Ref - https://github.com/Azure/azure-sdk-for-java/wiki/Test-Proxy-Migration#2-migrate-updated-recordings-to-assets-repohttps://github.com/Azure/azure-sdk-for-java/issues/33342